service/controller/resource/tccpnoutputs/create.go

```diff
@@ -72,39 +66,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	}
 
 	{
-		v, err := cloudFormation.GetOutputValue(outputs, DockerVolumeResourceNameKey)
-		if err != nil {
-			return microerror.Mask(err)
-		}
-		cc.Status.TenantCluster.TCCPN.DockerVolumeResourceName = v
-	}
-
-	if r.route53Enabled {
```

Comment:
why is this dropped?

Reply:
we weren't sure whether it was needed, @xh3b4sd can probably give more context.

Reply:
The initial version was not 100% so we got it straight. This is about the new TCCPN stack and there are only two outputs the stack defines. The rest does not exist so we dropped it.